shelley/chain-and-ledger/shelley-spec-ledger-test/shelley-spec-ledger-test.cabal

```diff
@@ -22,7 +22,9 @@ flag development
     manual: True
 
 library
-  hs-source-dirs:    src
+  hs-source-dirs:
+      src
+      test
```

Comment:
Hmm, not sure whether this is the best choice. But then again I'm not sure of a better one...

Reply:
Sharing source folders between components is something to avoid. Why is it needed?

Reply:
The original Shelley test package is nicely split into source/test - but with the intro of Shelley-ma-test, which needs to reuse chunks of shelley-test's test code, this neat separation got ruined (and this is the scar you're seeing here). We need to rethink how to arrange the various era test packages. I think we should combine all era tests into one package again.

Reply:
In that case, I would move the test code that needs to be reused by shelley-ma-test to the library of the shelley-test package.